iso_codes_info.h

```diff
@@ -0,0 +1,7 @@
+#ifndef ISO_CODES_INFO_H_
+#define ISO_CODES_INFO_H_
+
+extern const char *ISO_CODES_INFO;
+
+#endif /* ISO_CODES_INFO_H_ */
+
```

Comment:
This whole file seems to be unused, except for being included by `iso_codes_info.c.in`  which is also unused.